### PR TITLE
Improve MAVEN build Performance

### PR DESCRIPTION
**Pull request by ChenZhangg**
[Apache Maven Dependency Plugin](https://maven.apache.org/plugins/maven-dependency-plugin/index.html) can be used to find unused dependencies. And I found following list. Maybe we can remove them.
selenese-runner-java
{groupId='org.seleniumhq.selenium', artifactId='selenium-java'}
{groupId='org.slf4j', artifactId='jcl-over-slf4j'}
{groupId='org.glassfish.jaxb', artifactId='jaxb-runtime'}

According to [Maven parallel test](https://www.baeldung.com/maven-junit-parallel-tests), we can run tests in parallel.

=====================
If there are any inappropriate modifications in this PR, please give me a reply and I will change them.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -452,31 +457,16 @@ It supports test-case and test-suite which are Selenium IDE's native format.</de
...
         </configuration>
       </plugin>
     </plugins>
   </build>
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>org.seleniumhq.selenium</groupId>
-        <artifactId>selenium-java</artifactId>
-        <version>${selenium.version}</version>
```
Comment (vmi) [inline pom.xml:464]: Please do not delete this part and `</dependency>`.
If you remove this part, the version of selenium-java cannot be determined and a build error will occur.

File: pom.xml [hunk 2]
```diff
@@ -683,11 +669,7 @@ It supports test-case and test-suite which are Selenium IDE's native format.</de
       <artifactId>jakarta.xml.bind-api</artifactId>
       <version>2.3.3</version>
     </dependency>
-    <dependency>
-      <groupId>org.glassfish.jaxb</groupId>
-      <artifactId>jaxb-runtime</artifactId>
-      <version>2.3.4</version>
-    </dependency>
```
Comment (vmi) [inline pom.xml:690]: Please do not delete this part.
Removing this part will cause a runtime exception in Java 11 or later.

**Review by vmi — COMMENTED** [review]
Thanks for your pull request.
However, there are a few problems, please fix them.

**Comment by ChenZhangg** [comment]
ok the Maven Dependency Plugin is not accurate....

**Pull request merged**